tests/integration/check_subscriptions.rs

```diff
@@ -0,0 +1,201 @@
+use std::env;
+use std::fs::File;
+use std::io::Read;
+
+use lib::bigquery::client::{AccessTokenFromEnv, BQClient};
+use lib::check_subscriptions::fetch_and_process_new_subscriptions;
+use lib::models::aic::{AICModel, AIC};
+use lib::models::subscriptions::{Subscription, SubscriptionModel};
+use pretty_assertions::assert_eq;
+use serde::Deserialize;
+use serde_json::Value;
+use serial_test::serial;
+use time::{date, time, Format, OffsetDateTime};
+use uuid::{Uuid, Version};
+use wiremock::{matchers::any, Mock, MockServer, ResponseTemplate};
+
+use crate::utils::get_db_pool;
+
+#[derive(Debug, Deserialize)]
+pub struct AICSimple {
+    pub id: Uuid,
+    pub cj_event_value: String,
+    pub flow_id: String,
+    pub created: String,
+    pub expires: String,
+}
+
+fn get_aics_from_json(path: &str) -> Vec<AIC> {
+    let mut file = File::open(path).unwrap();
+    let mut data = String::new();
+    file.read_to_string(&mut data).unwrap();
+    let imported: Vec<AICSimple> =
+        serde_json::from_str(&data).expect("JSON was not well-formatted");
```

Comment:
Same comment here as elsewhere (re the error string)

Reply:
d233bc1